editor/block-mover/test/mover-label.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
+
+describe( 'block mover', () => {
+	const dir_up = -1,
```

Comment:
Variables should be named as camelCase, per guidelines:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#naming-conventions

Not sure if we really need the variables at all really, or at least not exclusively in the context of tests, since it doesn't really do anything to safeguard us against refactoring of the code under test.